syntax = "proto3";
package tutorial;

import "google/protobuf/timestamp.proto";

option java_package = "com.xy";
option java_outer_classname = "ExternalMeasurementDataPb";


message ExternalMeasurementDataBean {
    ParameterBody parameter = 1;
    PredictDataBody predictData = 2;
}

message PredictDataBody{
    string dataStr = 1;
}

message ParameterBody {
    string spatialEncoding  = 1; //空间编目
    string beginTime = 2; //起始时间
    string endTime = 3; //结束时间
    string deviceName = 4;//设备名称
}


